### PR TITLE
Fix bower-asset package problem

### PR DESCRIPTION
**Pull request by romulolink**
composer was displaying a package not found message for pushstream bower dependence.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,11 +13,17 @@
...
     "autoload": {
         "psr-4": {
             "dizews\\pushStream\\": ""
         }
+    },
+    "extra": {
+        "asset-installer-paths": {
+            "npm-asset-library": "vendor/npm",
+            "bower-asset-library": "vendor/bower"
+        }
```
Comment (dizews) [inline composer.json]: extra should be located in a project composer.json not in widget

File: composer.json [hunk 2]
```diff
@@ -13,11 +13,17 @@
     "require": {
         "yiisoft/yii2": "*",
         "guzzlehttp/guzzle": "~5.0",
-        "bower-asset/pushstream": "*"
+        "bower-asset/pushstream": "dev-master"
```
Comment (dizews) [inline composer.json:16]: ok

Comment (romulolink) [inline composer.json]: removed the changes on the composer package file.

**Comment by romulolink** [comment]
Removed changes on composer package file.

**Comment by dizews** [comment]
thanks!

**Pull request merged**